R/utils.R

```diff
@@ -54,4 +54,23 @@ validate_directed_graph_part <- function(g, type){
 validate_bidirected_graph_part <- function(g, type){
   validate_undirected(g, type)            # check if graph is undirected
   validate_simple(g, type)                # check if graph is simple
+}
+
```

Comment:
Let's add a comment here to tell the user the this function returns a list of Booleans

Reply:
Done. See #47